config.go

```diff
@@ -94,6 +96,8 @@ func New() *Frontman {
 		HTTPCheckTimeout:       15,
 		NetTCPTimeout:          3,
 		SSLCertExpiryThreshold: 7,
+		//TODO: Do we need this configurable?
+		SystemFields: []string{"unname", "os_kernel", "os_family", "os_arch", "cpu_model", "fqdn", "memory_total_B"},
```

Comment:
Looks like a typo(unname instead of uname) that came here from the frontman. Can you fix here?